### PR TITLE
Introduce attributes for "managing containers"

### PR DESCRIPTION
**Pull request by maximiliankolb**
In order to use the "managing container" section for multiple OS (e.g. importing a container image for Ubuntu), I want to introduce attributes to handle all OS-specific parameters.

Do you think the attributes should be part of the assembly or moved to the "attributes-base" file?

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)

### CODE REVIEW

**Review by adamlazik1 — COMMENTED** [review]
I agree with the use of the attributes.

Few thoughts:
1. The module is outdated, but that is definitely work for another PR:
2. Substitute values appear to be concrete rather than example so I'd leave the italics out.
3. There is also `Red Hat Container Catalog` mention on line 43. Not being able to comment on unchanged lines is sure annoying sometimes :sweat_smile:

**Comment by maximiliankolb** [comment]
Thanks Adam. I've applied your feedback. Please re-view.

File: guides/common/attributes-base.adoc [hunk 1]
```diff
@@ -111,3 +111,7 @@
 :SmartProxyServerTitle: Smart{nbsp}Proxy{nbsp}Server
 :Team: Foreman developers
 :sectanchors:
+:container-image-name: rhel7
+:container-product-name: Red Hat Container Catalog
+:container-repository-name: RHEL7
+:container-url: http://registry.access.redhat.com/
```
Comment (adamlazik1) [inline guides/common/attributes-base.adoc:117]: Please keep the alphabetical order.

**Review by adamlazik1 — COMMENTED** [review]
Please address point 2 from the previous [review](https://github.com/theforeman/foreman-documentation/pull/2056#pullrequestreview-1337101268).

File: guides/common/attributes-base.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/attributes-base.adoc]: Fixed.

**Comment by maximiliankolb** [comment]
Sorted in alphabetical order and removed `_`.

**Review by adamlazik1 — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   2c9302ee7..10d73a0a1  3.6 -> 3.6
   e2fbfd865..21d42ae72  3.5 -> 3.5
   5e11f9079..ab15a2fe4  3.4 -> 3.4
   12cd94724..eaf68d44f  3.3 -> 3.3

**Pull request merged**